data/KrishJ4856.json

```diff
@@ -3,22 +3,34 @@
   "displayStatsPublic": true,
   "type": "personal",
   "bio": "I am a high school student currently learning about Web Development and making an attempt to share my learning in public.",
-
+"avatar": "https://github.com/KrishJ4856.png",
```

Comment:
```suggestion
```

Reply:
Oh I didn't notice that! 😅 ... sorry my bad !

Reply:
Not a problem, I'll do it in a second